tests/run/cpp_nested_names_helper.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
Much as I like `#pragma once` in the real world, I'd rather use include guards for Cython tests. Just because they're standard